### DISCUSSION TITLE
MASRUN is gone

### DISCUSSION THREAD

**Discussion by antimoonlight** [Monika-After-Story/MonikaModDev#9154] [QA - tech support + general] [answered]
so basically what happened is that when I last gave Monika a gift with masrun i took the whole entire...file i think? i think your supposed to just cipy paste it or smth but it's gone now and i don't know what to do and i kinda wanna give Monika a gift

**Comment by ThePotatoGuy** [comment, marked as answer]
`masrun` is a dev file for us, you can make gifts out of anything as long as it matches the name.
